IEcho install instructions in README.md are incorrect.

Is this failing because I just installed .NET Core 3? 



`PS C:\Windows\system32> dotnet iecho install`


Could not execute because the specified command or file was not found.
Possible reasons for this include:
  * You misspelled a built-in dotnet command.
  * You intended to execute a .NET Core program, but dotnet-iecho does not exist.
  * You intended to run a global tool, but a dotnet-prefixed executable with this name could not be found on the PATH.
PS C:\Windows\system32> dotnet tool install -g Microsoft.Jupyter.Example.IEcho
error NU1101: Unable to find package microsoft.jupyter.example.iecho. No packages exist with this id in source(s): C:\Program Files\dotnet\sdk\NuGetFallbackFolder, Microsoft Visual Studio Offline Packages, nuget.org
The tool package could not be restored.
Tool 'microsoft.jupyter.example.iecho' failed to install. This failure may have been caused by:




* You are attempting to install a preview release and did not use the --version option to specify the version.
* A package by this name was found, but it was not a .NET Core tool.
* The required NuGet feed cannot be accessed, perhaps because of an Internet connection problem.
* You mistyped the name of the tool.

For more reasons, including package naming enforcement, visit https://aka.ms/failure-installing-tool

--- a/README.md
+++ b/README.md
@@ -8,8 +8,8 @@ This makes it easy to package, distribute, and install language kernels.
 For instance, the [**IEcho**](examples/echo-kernel/) sample kernel can be installed into a user's Jupyter environment with two commands:
 
 ```
-dotnet tool install -g Microsoft.Jupyter.Example.IEcho
-dotnet iecho install
+cd examples/echo-kernel/
+dotnet run -- install
 ```
 
 Once installed, the IEcho example can then be used like any other Jupyter kernel by running your favorite client:
@@ -17,6 +17,13 @@ Once installed, the IEcho example can then be used like any other Jupyter kernel
 ```
 jupyter notebook
 ```
+
+After a language kernel has been published as a NuGet package, it can be installed into a user's Jupyter environment with two commands. For example, if the IEcho kernel were published as a NuGet package named `Microsoft.Jupyter.Example.IEcho`, it could be installed via the following commands:
+
+```
+dotnet tool install -g Microsoft.Jupyter.Example.IEcho
+dotnet iecho install
+````
 
 ## Making New Language Kernels ##
 

